index.html

```diff
@@ -247,7 +248,7 @@ <h1 class="page-header">Dashboard</h1>
                 </div>
                 <div class="col-xs-9 text-right">
                   <div class="huge">0</div>
```

Comment:
This should be updated to match the chart on new dustiness data. If you want you can take care of it. Otherwise i will do it before merging. 😺